inst/shiny/modules/tab_nca/nca_setup.R

```diff
@@ -224,7 +226,7 @@ nca_setup_server <- function(id, data, adnca_data) { # nolint : TODO: complexity
       updateSelectInput(
         session,
         inputId = "select_dosno",
-        label = "Choose the Dose Number:",
+        label = "Choose the profiles for the NCA:",
         choices = rows_for_selected_analytes$DOSNO,
```

Comment:
question: Shouldn't `NCA_PROFILE` be referenced here as well?
```suggestion
        choices = rows_for_selected_analytes$NCA_PROFILE,
```